plugins/filters/app_janus/filter_app_janus_tracer.js

```diff
@@ -1455,7 +1462,7 @@ function ContextManager (self, tracerName, lru) {
       })
     }
 
-    mediaMetrics.streams = null
+    //mediaMetrics.streams = null
```

Comment:
This can be removed, I did not see a difference in memory management with this call